dotCMS/src/main/java/com/dotcms/rest/api/v1/authentication/ApiTokenResource.java

```diff
@@ -196,7 +207,68 @@ public final Response issueApiToken(@Context final HttpServletRequest request,
         return Response.ok(new ResponseEntityView(map("token", token,"jwt", jwt), EMPTY_MAP)).build(); // 200
     }
 
-    
+    /**
+     * Request a Token to a remote server
+     * @param request
+     * @param response
+     * @param formData - json data, expecting
+     * <pre>
+     * {
+     *    token: {
+     *      netmask:'192.168.1.0/24',
+     *      expirationDays:1000,
+     *      userId:'dotcms.org.1'
+     *      claims: {
+     *          label: 'Push Publish'
+     *      }
+     *    },
+     * 	  remote: {
+     * 	    host: localhost,
+     * 		port: 8090
+     * 	  },
+     * 	  auth: {
+     * 	    login: admin@dotcms.com,
+     * 		pasword: [password in Base64]
+     * 	  }
+     * }
+     * </pre>
+     * @return Response
+     */
+    @PUT
+    @Path("/_remote")
```

Comment:
This shouldn't have the `_` prefix, because it's not an operation.

Reply:
done https://github.com/dotCMS/core/pull/19643/commits/0c8f8477e3a1c3d9654944171e3904c4f27b06e0#diff-63fbac6d5f8fa5e3ac1967d35356f34c1bc23b815139fb0d40c17398ac3d5554R234